src/full/Agda/Utils/Maybe/Strict.hs

```diff
@@ -1,3 +1,9 @@
+-- Liang-Ting Chen (04-07-2019):
+-- Consider using Data.Maybe.Strict instead
+-- Andreas Abel (05-05-2019)@GitHub:
```

Comment:
You meant 5th July 2019, I suppose?  
Date stamps in the Agda code base follow the Swedish date format: YYYY-MM-DD. :)

Reply:
Thanks! Done.